director/advertise.go

```diff
@@ -177,10 +204,17 @@ func AdvertiseOSDF() error {
 			write = false
 		}
 
+		listings := false
+		if ns.DirlistHost != "" {
+			listings = true
+		}
+
 		caps := server_structs.Capabilities{
 			PublicReads: !ns.UseTokenOnRead,
 			Reads:       ns.ReadHTTPS,
 			Writes:      write,
+			Listings:    listings,
+			DirectReads: true, // Topology namespaces should probably always have this turned on
```

Comment:
There's no way to configure a similar feature through topology, so we need to pick a default (true/false). I think we want to go with the more permissive, since this doesn't affect general security.

Reply:
That's kinda what I was thinking, makes sense! Just wanted to know what your thought was for it